src/asmcnc/apps/drywall_cutter_app/config/config_loader.py

```diff
@@ -201,6 +238,12 @@ def fix_config(config_name):
             config_classes.Configuration.__init__
         ).args[1:]
 
+        try:
+            # Get the version of the configuration file
+            config_version = cfg["version"]
```

Comment:
config_version = cfg.get("version", "1.0")

Removes the requirement for the try except and will be neater for future code.